src/lovelace/settings.py

```diff
@@ -0,0 +1,126 @@
+"""
+Django settings for lovelace project.
+
+Generated by 'django-admin startproject' using Django 1.11.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.11/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.11/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'u5s4$^ta#=#j9+mno!87@+h!$8u6hx^pi#mfi-f31)i&c6)w^b'
```

Comment:
Guess we'll change this once the repo goes fully private? Can you change this down the line?

If you want to keep the repo public maybe you can just add this file to .gitignore but maybe that doesn't make sense.